.github/workflows/ci.yml

```
@@ -127,10 +127,16 @@ jobs:
         uses: reanahub/reana-github-actions/local-run@v1
         with:
           commands: |
+            # Install ROOT dependencies
+            sudo apt-get update -y
+            sudo apt-get install binutils cmake dpkg-dev g++ gcc libssl-dev git libx11-dev libxext-dev libxft-dev libxpm-dev python3 libtbb-dev libvdt-dev libgif-dev
+
             # Install ROOT
-            wget https://root.cern/download/root_v6.24.02.Linux-ubuntu20-x86_64-gcc9.3.tar.gz
-            tar -xzvf root_v6.24.02.Linux-ubuntu20-x86_64-gcc9.3.tar.gz
+            # ROOT is needed as Snakemake does not run the workflow steps in containers
+            wget https://root.cern/download/root_v6.32.04.Linux-ubuntu24.04-x86_64-gcc13.2.tar.gz
```

```suggestion
            wget -q https://root.cern/download/root_v6.32.04.Linux-ubuntu24.04-x86_64-gcc13.2.tar.gz
```

(in order to save many unnecessary lines in the job logs)